src/App.js

```diff
@@ -35,6 +36,16 @@ class App extends React.Component {
     ));
   }
 
+  removeProductFromCart(id) {
+    this.setState(({ cartProducts }) => {
+      const indexOfProduct = cartProducts
```

Comment:
Por algum motivo voltou pra aquele probleminha de antes. Ja vou corrigir logo ^^

```suggestion
      const indexOfProduct = cartProducts.reverse()
```